### ISSUE TITLE
stopContainers = false => containers that do not exist are not created

### ISSUE THREAD

**Issue by cer** [avast/gradle-docker-compose-plugin#237]
I have the following configuration:

https://github.com/eventuate-tram/eventuate-tram-examples-customers-and-orders/blob/e94501afbce131e20f84bf3d57b985f4bc96fe82/build.gradle#L32-L49

https://github.com/eventuate-tram/eventuate-tram-examples-customers-and-orders/blob/e94501afbce131e20f84bf3d57b985f4bc96fe82/build.gradle#L125

if I run the following commands:

```
./gradlew mysqlbinloginfrastructureComposeUp
./gradlew mysqlbinlogComposeUp
```

then I see

```
./gradlew mysqlbinlogComposeUp

> Task :mysqlbinloginfrastructureComposeUp
Cached services infos loaded while 'stopContainers' is set to 'false'.

> Task :mysqlbinlogComposeUp
Cached services infos loaded while 'stopContainers' is set to 'false'.
```

and the only the containers started by `mysqlbinloginfrastructureComposeUp` are created.

**Comment by augi** [comment]
The problem is that you set `projectName` to `null` so Docker Compose supposes that both configurations are the same _project_.

**Comment by cer** [comment]
A couple of points.

First, we had set `stopContainers=false` because if a container failed to start the plugin was stopping the containers that were running and removing them which hindered debugging.

Second, mysqlbinlogCompose is a super-set of mysqlbinloginfrastructure. This was done for a couple of reasons. Different tests need a different subset of the containers. Also, it can accelerate startup of the  mysqlbinlogCompose to do it in two phases. First, mysqlbinloginfrastructure and then mysqlbinlogCompose. Consequently, they are the same 'project', and using different project names won't work as far as I know.

**Comment by augi** [comment]
I understand it now, thanks for your explanation. Unfortunately, this plugin probably doesn't support your use-case.

**Comment by cer** [comment]
ic. BTW it works fine with `stopContainers=true`

**Comment by augi** [comment]
I will try to reproduce your issues and prepare a PR.

**Comment by SJrX** [comment]
@augi , did that commit fix this issue and did that make it in to 0.12.1, I ended up having a similar setup as the reporter, and even though I'm using 0.12.1 the behaviour is still there.

**Comment by augi** [comment]
The bug described in this issue was solved in `0.12.1`.
